### PR TITLE
Fix FAQ, add projects to /namnder and fix /namnder

### PR DESCRIPTION
**Pull request by olofbmelcherson**
- [X] I have performed a self-review of my changes
  - Instructions for easily viewing your changes locally is available in [the readme](https://github.com/datasektionen/bawang-content?tab=readme-ov-file#k%C3%B6ra-lokalt).
- [X] I have updated both Swedish and English pages (if not motivate why)

### CODE REVIEW

**Comment by olofbmelcherson** [comment]
The commit "Updated pretty much every tab under /en/chapter and some parts of /sektionen" has already been merged but i accidentaly included it :(

File: en/clubs/dåre/sidebar.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Contact
+
+Do you want to contact the project leader of dÅre? Send a mail to [dare@datasektionen.se](mailto:dare@datasektionen.se).
```
Comment (Benjaneb) [inline en/clubs/dåre/sidebar.md:3]:
```suggestion

```

File: en/clubs/projectpride/body.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Project Pride is the chapters annual float in the Stockholm Pride Parade
```
Comment (Benjaneb) [inline en/clubs/projectpride/body.md:1]:
```suggestion
Project Pride is the chapter's annual float in the Stockholm Pride Parade.
```

File: en/clubs/projectpride/sidebar.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## Contact
+
+Do you want to contact Prioject Pride? Send a mail to [pride@datasektionen.se](mailto:pride@datasektionen.se).
```
Comment (Benjaneb) [inline en/clubs/projectpride/sidebar.md:3]:
```suggestion

```

File: en/clubs/projectpride/sidebar.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+#### DESCtop
```
Comment (Benjaneb) [inline en/clubs/projectpride/sidebar.md:5]:
```suggestion
#### Project leader
```

File: en/clubs/studs/body.md [hunk 5]
```diff
@@ -0,0 +1 @@
+[Studs](https://studieresan.se) - The Study Visit, a course organised by students of the Chapter
```
Comment (Benjaneb) [inline en/clubs/studs/body.md:1]:
```suggestion
[Studs](https://studieresan.se) - The Study Visit, a course organised by students of the Chapter.
```

File: en/clubs/studs/sidebar.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+## Contact
+
+Do you want to contact the Project Leaders of STUDS? Send a mail to [ansvarig@studs.se](mailto:ansvarig@studs.se).
```
Comment (Benjaneb) [inline en/clubs/studs/sidebar.md:3]:
```suggestion

```

File: en/clubs/vårbalen/body.md [hunk 7]
```diff
@@ -0,0 +1 @@
+Vårbalen is the chapters annual spring ball.
```
Comment (Benjaneb) [inline en/clubs/vårbalen/body.md:1]:
```suggestion
Vårbalen is the chapter's annual spring ball.
```

File: en/clubs/vårbalen/sidebar.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+## Contact
+
+Do you want to contact the Spring Ball? Send a mail to [varbal@datasektionen.se](mailto:varbal@datasektionen.se)
```
Comment (Benjaneb) [inline en/clubs/vårbalen/sidebar.md:3]:
```suggestion

```

File: namnder/dåre/meta.toml [hunk 9]
```diff
@@ -0,0 +1,3 @@
+Image = "Image = "https://stacken.är.den.verkligen.uppe/a.png""
```
Comment (Benjaneb) [inline namnder/dåre/meta.toml:1]:
```suggestion
Image = "https://stacken.är.den.verkligen.uppe/a.png"
```

File: namnder/dåre/sidebar.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
+## Kontakt
+
+#### Projektledare för dÅre
+
+karl Lindblad </br>
```
Comment (Benjaneb) [inline namnder/dåre/sidebar.md:5]:
```suggestion
Karl Lindblad </br>
```

File: namnder/metaspexet/meta.toml [hunk 11]
```diff
@@ -0,0 +1,3 @@
+Image = "Image = "https://stacken.är.den.verkligen.uppe/a.png""
```
Comment (Benjaneb) [inline namnder/metaspexet/meta.toml:1]:
```suggestion
Image = "https://stacken.är.den.verkligen.uppe/a.png"
```

File: namnder/projectpride/meta.toml [hunk 12]
```diff
@@ -0,0 +1,3 @@
+Image = "Image = "https://stacken.är.den.verkligen.uppe/a.png""
```
Comment (Benjaneb) [inline namnder/projectpride/meta.toml:1]:
```suggestion
Image = "https://stacken.är.den.verkligen.uppe/a.png"
```

File: namnder/studs/meta.toml [hunk 13]
```diff
@@ -0,0 +1,3 @@
+Image = "Image = "https://stacken.är.den.verkligen.uppe/a.png""
```
Comment (Benjaneb) [inline namnder/studs/meta.toml:1]:
```suggestion
Image = "https://stacken.är.den.verkligen.uppe/a.png"
```

File: namnder/varbalen/meta.toml [hunk 14]
```diff
@@ -0,0 +1,3 @@
+Image = "Image = "https://stacken.är.den.verkligen.uppe/a.png""
```
Comment (Benjaneb) [inline namnder/varbalen/meta.toml:1]:
```suggestion
Image = "https://stacken.är.den.verkligen.uppe/a.png"
```

File: sektionen/faq/body.md [hunk 15]
```diff
@@ -1,104 +1,26 @@
...
-SM, där alla sektionens medlemmar får närvara. Skicka en motion till
-styrelsen inför det DM som brukar hållas precis innan SM där du
-motiverar dig. Tveka inte att höra av dig om du har frågor om formalia
-eller vill komma i kontakt med någon du kan bolla idéer med - detta är
-högst uppskattat. Därefter uppmanas du närvara på DM innan SM för
-diskussion kring motionen och till SM följs detta sedan upp av ett
-motionssvar författat av d-Rek. En LaTeX-mall för motioner hittar du
-under rubriken "Motion" på [denna](/sektionen/organisation) sida.
+### Vem skriver jag till om jag har några fler frågor?
+[Info@datasektionen.se](mailto:info@datasektionen.se)
```
Comment (Benjaneb) [inline sektionen/faq/body.md:26]:
```suggestion
[info@datasektionen.se](mailto:info@datasektionen.se)
```

File: studier/faq/body.md [hunk 16]
```diff
@@ -1,3 +1,3 @@
 
 # Vanliga frågor
-Svar på vanliga frågor går att hitta på [THS hemsida](https://ths.kth.se/sv/help)
+Svar på vanliga frågor går att hitta på [THS hemsida](https://thskth.se/sv/utbildning)
```
Comment (Benjaneb) [inline studier/faq/body.md:3]:
```suggestion
Svar på vanliga frågor går att hitta på [THS hemsida](https://thskth.se/sv/utbildning).
```

File: sektionen/faq/body.md [hunk 17]
```diff
@@ -1,104 +1,26 @@
...
-## FAQ angående fester, arrangemang och events
+### Hur blir jag medlem?
+Du blir medlem genom att studera vid ett av programmen som tillhör sektionen och att betalat din [medlemsavgift till THS](http://ths.kth.se/membership).
 
+### Vad kan jag göra om jag inte är medlem?
+Du kan alltid sitta och studera i vår sektionslokal META (Osquars Backe 21) mellan 6-17, men för att ta del i de event som sektionen anordnar eller vara med i dess nämnder så måste du vara medlem.
 
-### Jag tycker det vore asfett att göra en fest, men jag vet inte hur..
+### Hur lär jag mig vad som händer på sektionen?
+Alla evenemang som sektionen anordnas marknadsförs på hemsidan! De hamnar i vårt vanliga nyhetsflöde men vill du endast se event som anordnas och till och med en kalender över dessa kan du gå till [Event](https://datasektionen.se/nyheter?itemType=POST).
```
Comment (Benjaneb) [inline sektionen/faq/body.md:10]:
```suggestion
Alla evenemang som sektionen anordnas marknadsförs på hemsidan! De hamnar i vårt vanliga nyhetsflöde men vill du endast se event som anordnas kan du gå till [Event](https://datasektionen.se/nyheter?itemType=EVENT) där det också finns en kalender över dessa.
```

**Review by Benjaneb — CHANGES_REQUESTED** [review]

**Comment by olofbmelcherson** [comment]
All changes accepted

**Review by Benjaneb — APPROVED** [review]
69 st ändringsförfrågningar kanske inte var den bästa idéen, vilken jävla commit history :rofl:

**Pull request merged**